subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1310,6 +1336,14 @@ u8_t scan_rsp_set(struct ll_adv_set *adv, u8_t len, u8_t const *const data)
 
 static int init_reset(void)
 {
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
```

Comment:
Remove this code block, to be implemented when Max_Extended_Advertising_Events parameter and duration parameter is implemented in HCI_LE_Set_Extended_Advertising_Enable.